content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,498 @@
+---
+title: Federated Authentication to Role Mapping API
+kind: documentation
+beta: true
+further_reading:
+- link: "account_management/rbac/log_management/"
+  tag: "Documentation"
+  text: "RBAC for Log Management"
+---
+
+<div class="alert alert-warning">
+Ask your customer success manager or <a href="/help">Datadog Support</a> to enable this feature.
+</div>
+
+If you are using Federated Authentication mechanisms, this API allows you to automatically map groups of users to roles in Datadog.
+
+**Note**: If you are a SAML user, and you have been using the existing beta Federated Mapping mechanism (`roles_v2_saml`), Datadog strongly recommends that you transition to using this API.
+
+## Requests
+
+All the API endpoints below can have two different host endpoints:
+
+* If you are on the Datadog US site: `https://api.datadoghq.com/api/`
+* If you are on the Datadog EU site: `https://api.datadoghq.eu/api/`
+
+### Create a new Authentication mapping
+
+
+Create a new AuthN Mapping from a JSON body. Returns the newly created AuthN Mapping.
+
+| Method | Endpoint path | Required payload |
+|--------|--------------|------------------|
+| `POST`  | `/v2/authn_mappings`  | JSON       |
+
+##### ARGUMENTS
+
+* **`id`** [*required*, *default*=none]:
+  The UUID of the `authinmap` being updated. For more information about finding your UUID, see the [Role API documentation][1].
```

Comment:
`id` is not expected for create request payloads and passing it in will actually return an error